packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md

```diff
@@ -155,6 +157,57 @@ https://app.scaleft.com/v1/teams/${team_name}/servers
 	]
 }
 ```
+### Update a Server
+
+<ApiOperation method="PUT" url="https://app.scaleft.com/v1/teams/${team_name}/servers/${server_id}" />
+Update a Server. This endpoint is only available with the PolicySync feature, which is currently in EA
+
+This endpoint requires one of the following roles: `access_admin`, or `server_admin`.
+
+#### Request path parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| `server_id`   | string | The UUID of the Server |
+| `team_name`   | string | The name of your Team |
+
+
+#### Request query parameters
+
+This endpoint has no query parameters.
+
+#### Request body
+
+
+This endpoint requires an object with the following fields.
+| Properties | Type        | Description          |
+|----------|-------------|----------------------|
+| `labels`   | object | (Optional) A map of key value pairs. These labels will overwrite all labels previously supplied via the API for this server. Labels from any other source can only be updated using that source. The prefix 'api.' will be automatically prepended if not supplied. |
```

Comment:
```suggestion
| `labels`   | object | (Optional) A map of key value pairs. These labels overwrite all labels previously supplied through the API for this server. You can only update labels from other sources using that source. If you don't supply the prefix 'api.', it is automatically prepended. |
```

Reply:
@susanharper-okta thanks for the review. I've addressed your comments in 3f3fbd5